<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gsmscale</class>
 <widget class="QMainWindow" name="gsmscale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>455</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gsmscale</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="CB_PageSize">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_CB_PageSize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select page size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_TXT_Length">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_TXT_Width">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_X">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif Telugu Light</family>
      <pointsize>36</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="TXT_CUR_Weight">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>371</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	text-align: left;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_CUR_Weight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Weight (g)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_GSM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>371</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber {
	color:rgb(0, 170, 0);
	background:black;
	text-align:center;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="LAB_GSM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>GSM</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SPIN_Length">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>151</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SPIN_Width">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>151</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInplicare_GSM_Calculator">
    <property name="title">
     <string>Inplicare GSM Calculator</string>
    </property>
   </widget>
   <addaction name="menuInplicare_GSM_Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>CB_PageSize</tabstop>
  <tabstop>TXT_CUR_Weight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
